packages/graphql-language-service-server/src/__tests__/MessageProcessor-test.ts

```diff
@@ -295,7 +296,7 @@ describe('MessageProcessor', () => {
     };
 
     const result = await messageProcessor.handleDefinitionRequest(test);
-    await expect(result[0].uri).toEqual(`file://${queryDir}/test3.graphql`);
```

Comment:
hmmm... I don't think LSP will like this. LSP (and thus vscode, and other LSP implementations) need this result to be a URL

Reply:
I had changed the value of queryDir to be an uri. To reflect that butter Ive now renamed that variable to queryPathUri instead.